relayer/msgs.go

```diff
@@ -363,22 +491,26 @@ func (c *Chain) MsgRelayAcknowledgement(counterparty *Chain, packet *relayMsgPac
 		return
 	}
 
-	version := clienttypes.ParseChainID(counterparty.ChainID)
+	if ackRes == nil {
+		return nil, fmt.Errorf("ack packet [%s]seq{%d} has no associated proofs", counterparty.ChainID, packet.seq)
+	}
+
+	version := clienttypes.ParseChainID(c.ChainID)
```

Comment:
this should be counterparty.ChainID. We are submitting a proof of an acknowledgement on the original sender chain so the counterparty is the chain who we specified the timeout for